php/libraries/Visit.class.inc

```diff
@@ -0,0 +1,220 @@
+<?php
+/**
+ * This file contains the Visit class.
+ *
+ * PHP Version 5
+ *
+ * @category Main
+ * @package  Main
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+/**
+ * Base class that contains method to access the visit of projects
```

Comment:
I think a class named "Visit" should be a model of a Visit, not a class to access visits.

Reply:
I do agree that this class does not represent a visit however i would argue that for this PR it would be sufficent to change the name of the class to visitController and in a subsequent PR establish a clear class architecture that follows the upcoming DAO model